07-reproj.qmd

```diff
@@ -37,9 +38,47 @@ world = gpd.read_file('data/world.gpkg')
 
 ## Introduction
 
+@sec-coordinate-reference-systems introduced coordinate reference systems (CRSs), with a focus on the two major types: geographic ('lon/lat', with units in degrees longitude and latitude) and projected (typically with units of meters from a datum) coordinate systems. This chapter builds on that knowledge and goes further. It demonstrates how to set and transform geographic data from one CRS to another and, furthermore, highlights specific issues that can arise due to ignoring CRSs that you should be aware of, especially if your data is stored with lon/lat coordinates. 
```

Comment:
This looks like really good, lucid, concise prose. Our readers will benefit greatly.

Reply:
Not sure how close the prose is to geocompr, but reads well to me!